source/features/conventional-commits.tsx

```diff
@@ -7,65 +7,42 @@ import observe from '../helpers/selector-observer.js';
 import {commitTitleInLists} from '../github-helpers/selectors.js';
 import {parseConventionalCommit, removeCommitAndScope} from '../helpers/conventional-commits.js';
 
-const types = new Map([
-	['feat', 'Feature'],
-	['fix', 'Fix'],
-	['chore', 'Chore'],
-	['docs', 'Docs'],
-	['build', 'Build'],
-	['refactor', 'Refactor'],
-	['test', 'Test'],
-	['ci', 'CI'],
-	['perf', 'Performance'],
-]);
-
-function createLabelElement(type: string, scope?: string): JSX.Element {
-	const label = types.get(type)!;
-
-	return (
-		<span className={`IssueLabel hx_IssueLabel rgh-commit-type-label rgh-commit-type-label-${type}-colors`}>
-			{scope ? `${label}: ${scope}` : label}
-		</span>
-	);
-}
-
 function renderLabelInCommitTitle(commitTitleElement: HTMLElement): void {
-	const match = parseConventionalCommit(commitTitleElement.textContent);
+	const textNode = commitTitleElement.firstChild!;
+	const commit = parseConventionalCommit(textNode.textContent);
 
-	const {type, scope} = match?.groups ?? {};
-	if (!type || !types.has(type)) {
+	if (!commit) {
 		return;
 	}
 
-	commitTitleElement.prepend(createLabelElement(type, scope));
-	removeCommitAndScope(commitTitleElement, match!);
-}
+	commitTitleElement.prepend(
+		<span className="IssueLabel hx_IssueLabel mr-2" rgh-conventional-commits={commit.rawType}>
+			{commit.type}
+		</span>,
+		commit.scope ? <em>{commit.scope}</em> : '',
+	);
 
-function initRepoCommitList(signal: AbortSignal): void {
-	observe(`:is(${commitTitleInLists}) h4 > span`, renderLabelInCommitTitle, {signal});
+	removeCommitAndScope(textNode);
 }
 
-function initPrCommitList(signal: AbortSignal): void {
-	observe(`:is(${commitTitleInLists}) a`, renderLabelInCommitTitle, {signal});
+function init(signal: AbortSignal): void {
+	observe(`:is(${commitTitleInLists}) h4 > span > a:first-child`, renderLabelInCommitTitle, {signal});
```

Comment:
I don't know why this was split between PR and Repo commit lists, the DOM was identical and this works on both.